tests/ballerina-spec-conformance-tests/S05_builtin_abstract_types/tests/deviations/listener.bal

```diff
@@ -1,66 +0,0 @@
-// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Instead of removing this, shall we move it as a conforming test?

Reply:
Will move to tests/ballerina-spec-conformance-tests/S05_builtin_abstract_types/tests/ with expected  __attach signature.